### ISSUE TITLE
Conflictive with "WME Tab Manager" Script.

### ISSUE THREAD

**Issue by santy2001** [TheCre8r/WME-BackEnd-Data#5]
Hi,

When this script is working it seems like it changes the icons of the "WME Tab Manager" script and also makes some of them to not display correctly or simply not appear.
<img width="1280" alt="captura de pantalla 2018-12-31 a la s 10 04 50 a m" src="https://user-images.githubusercontent.com/17103864/50562680-0eaff480-0ce4-11e9-9940-3cb6442c7b2c.png">

CONSOLE DEBUG(between two scripts):
```
env: schedule "WME Tab Manager" for document-idle
env: execute script WME Tab Manager @ the unsafe context now!
WME Tab Manager: Tabs found, enabling observers
WMEBED:  bootstrap attempt 1
WMEBED:  santyg2001
WMEBED:  Initializing
WMEBED:  CSS Injected
WMEBED:  Settings Loaded
WMEBED:  Update Alert Ran
WMEBED:  #WMEBED-Debug is false
WMEBED:  Settings Initialized
WMEBED:  Settings Saved {“AdPin”:false,“Debug”:false,“lastVersion”:“2018.12.30.00"}
WMEBED:  Tab Initialized
```

**Comment by TheCre8r** [comment]
Issue has been isolated. On the roadmap to be fixed.

**Comment by TheCre8r** [comment]
Fixed in [b0e78e4](https://github.com/TheCre8r/WME-BackEnd-Data/commit/b0e78e4640d94cd2b32ca1f106eeb5881e4542cb#diff-11889ab02876849eaf9df26ada5c7ffa)
